05/app.js

```diff
@@ -0,0 +1,18 @@
+import ReactDOM from 'react-dom/client'
+import React from "react";
+import PropTypes from 'prop-types'
+import { createRoot } from 'react-dom/client';
+import Menu from './components/Menu';
+import Nav from './components/Nav';
+import MenuItem from '../03/MenuItem';
```

Comment:
Importujemy tylko elementy, z których korzystamy - tutaj będzie to `createRoot` oraz `Menu` (po zmianach opisanych niżej)